src/index.js

```diff
@@ -73,6 +80,14 @@ function renderHook(callback, { initialProps, wrapper } = {}) {
   })
   const { unmount, update } = testRenderer
 
+  function unmountHook() {
```

Comment:
what happens if someone calls this then then `cleanup`?  does the second `unmount` fail?  should `unmountHook` also remove the callback from `unmounts` (making [this line](https://github.com/testing-library/react-hooks-testing-library/pull/106/files#r296980396) no longer necessary at all)

Reply:
I see there is a test around this case.  I personally feel that having `unmountHook` remove the callback from the `unmounts` array is bit clearer to what is happening rather than relying on the behaviour of `unmount`, but I don't care enough to block this for it.

Reply:
I agree with you, i will fix it.